sandbox/sandbox/settings.py

```diff
@@ -53,7 +54,7 @@
     "wagtail.search",
     "wagtail.admin",
     "wagtail",
-    "wagtail.contrib.modeladmin",
+    "wagtail_modeladmin" if WAGTAIL_VERSION >= (5, 1) else "wagtail.contrib.modeladmin",
     "wagtailfontawesome",
```

Comment:
I wonder if we have to update this to `wagtailfontawesomesvg` instead

Reply:
I don't know for sure. It makes sense though. Can you test it out using the sandbox app?